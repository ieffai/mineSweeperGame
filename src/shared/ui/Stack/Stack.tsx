import { Stack as MuiStack, type StackProps as MuiStackProps } from '@mui/material';

export { MuiStack as Stack };
export type { MuiStackProps as StackProps };
