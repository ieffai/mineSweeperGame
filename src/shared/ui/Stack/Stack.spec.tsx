import { render } from '@testing-library/react';
import { Stack } from './Stack';

describe('Stack', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<Stack />);
    expect(asFragment()).toMatchSnapshot();
  });
});
