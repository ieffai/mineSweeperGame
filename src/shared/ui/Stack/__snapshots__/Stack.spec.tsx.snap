// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Stack > should render successfully 1`] = `
<DocumentFragment>
  <div
    class="MuiStack-root css-nen11g-MuiStack-root"
  />
</DocumentFragment>
`;
